doc/source/reference/random/performance.rst

```diff
@@ -72,3 +72,85 @@ performance was computed using a geometric mean.
 .. note::
 
    All timings were taken using Linux on a i5-3570 processor.
+
+Performance on different Operating Systems
+******************************************
+Performance differs across platforms due to compiler and hardware availability
+(e.g., register width) differences. The default bit generator has been chosen
+to perform well on 64-bit platforms.  Performance on 32-bit operating systems
+is very different.
+
+The values reported are the time in ns per random value. The performance index
+is a normalized score relative to the performance of MT19937 in each table.
```

Comment:
When making benchmark tables, it's necessary to state whether lower is better or higher is better. That the performance index has its direction switched from the raw timings makes things difficult. :-)

Reply:
I have turned everything into a score now, where higher is better.